engine/polyglot-api/src/test/scala/org/enso/polyglot/ModuleManagementTest.scala

```diff
@@ -176,6 +176,9 @@ class ModuleManagementTest extends AnyFlatSpec with Matchers {
     )
     val exception =
       the[PolyglotException] thrownBy mod2.getAssociatedConstructor
-    exception.getMessage shouldEqual "Module_Does_Not_Exist"
+    exception.getMessage shouldEqual "org.enso.compiler.exception." +
```

Comment:
This is bad – this code path should use normal compilation, so we probably should get a compile error, not an internal error. Please open an issue regarding fixing this (unless the fix is trivial).